### ISSUE TITLE
fn build should include contents of directory with non-dockerfile (ie language runtimes) funcs

### ISSUE THREAD

**Issue by carimura** [fnproject/cli#146]
for example, if my runtime is Go, and then I include a template file in the func dir and I want to open it, parse it, and return it to stdout, that template file is not included in the way the CLI does the build. Seems to me like the build should include everything in the current directory.

I can of course switch to runtime Docker, but then I'm dealing with Docker and that's not ideal. I want the simplicity of the default deal.

**Comment by zootalures** [comment]
If this is done by default it will tend to bloat out function images (especially with functions with source and binaries) 

Could this be done via a stanza in func.yaml? 
```
includes: 
   - *.html
   -  my/local/dir:container/dir
```
(following docker copy conventions or similar)

**Comment by rdallman** [comment]
I agree, we shouldn't shove everything in by default (bloat). we do some magic with the cli to minimize build size already. 

templates are always a pain to have to carry around, this is a specific case. +1 to have `includes:` to handle this, surprised we didn't have that already.

+2 it is then

**Comment by treeder** [comment]
+1 on the includes directive.

**Comment by carimura** [comment]
Includes works for me.
